test/tools/staking_genesis.json

```diff
@@ -1,15 +1,16 @@
 {
     "params": {
-      "max_allowances": 10000,
```

Comment:
Oh, you also use formatter for JSON, nice. Can you add linting and formatting to fmt/lint rules in package.json too?

Reply:
Oh yeah i think that's just [prettier](https://github.com/prettier/prettier) that i have set in my editor. Yeah i'll look into setting it up.

Reply:
done